components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests

```diff
@@ -150,11 +164,26 @@ function cleanup_kfp() {
 }
 
 if [[ -z "${EKS_EXISTING_CLUSTER}" ]]; then
-  launch_eks
+  # Launch all of these in parallel to reduce start-up time
+  EKS_CLUSTER_NAME="${DEPLOY_NAME}-eks-cluster"
```

Comment:
why did this move?

Reply:
We want to launch EKS at the same time as creating FSx because both operations take a long time. However, when we use the `&` operator, it starts the shell in a different process and therefore any variables set in that shell are not set in the caller shell. So I bypass this by setting the name of the cluster before calling.